ffi/png.lua

```diff
@@ -30,11 +30,11 @@ function Png.encodeToFile(filename, mem, w, h)
     end
 end
 
-function Png.decodeFromFile(filename)
+function Png.decodeFromFile(filename, fmt)
     local width = ffi.new("int[1]")
     local height = ffi.new("int[1]")
     local ptr = ffi.new("unsigned char*[1]")
-    local err = lodepng.lodepng_decode32_file(ptr, width, height, filename)
+    local err = lodepng.lodepng_decode_file(ptr, width, height, filename, fmt, 8)
```

Comment:
What's that 8 at the end? (It sounds suspiciously low compared to 32 for color screens, at a quick glance without looking up the API/headers. :-P)

Reply:
That's the *per channel* bitdepth ;).

(i.e., bpp = ncomp * 8).

Reply:
Because we're definitely not doing HDR ;).

Reply:
Aww, 10 bit ftw! :-P